x-pack/legacy/plugins/uptime/public/lib/helper/__tests__/stringify_url_params.test.ts

```diff
@@ -20,4 +20,26 @@ describe('stringifyUrlParams', () => {
     });
     expect(result).toMatchSnapshot();
   });
+
+  it('creates expected string value when ignore empty is true', () => {
+    const result = stringifyUrlParams(
+      {
+        autorefreshInterval: 50000,
+        autorefreshIsPaused: false,
+        dateRangeStart: 'now-15m',
+        dateRangeEnd: 'now',
+        filters: 'monitor.id: bar',
+        search: undefined,
+        selectedPingStatus: undefined,
+        statusFilter: '',
+        pagination: undefined,
+      },
+      true
+    );
+    expect(result).toMatchSnapshot();
```

Comment:
I'm have no issue with using a snapshot here because the end result is the same but @andrewvc and I have discussed that for simpler shapes/assertions it might be cleaner to perform an `expect` against a hardcoded value/string so the reviewer/reader can simply see the expected result without looking in another file. I don't think we mentioned it to you until now though.

Reply:
i did added some simple checks. Below, but in any case i think its good to have snapshot test along.